backend/base/models.py

```diff
@@ -357,6 +357,10 @@ def clean(self):
         super().clean()
         if not self.timestamp:
             self.timestamp = datetime.now()
+        if self.type == "AA" or self.type == "BI" or type == "VE":
```

Comment:
Why not `self.type == "OP"` as well?

Reply:
You can have multiple global remarks (=OP) for a building, but you arrive, go inside & leave only once at a building.